dotnet-testing/MockingIOptions/MockingIOptions/Program.cs

```diff
@@ -0,0 +1,37 @@
+using MockingIOptions.Configuration;
+using MockingIOptions.Services;
+
+namespace MockingIOptions
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            var builder = WebApplication.CreateBuilder(args);
+
+            // Add services to the container.
+            builder.Services.AddControllers();
+            builder.Services.AddEndpointsApiExplorer();
+            builder.Services.AddSwaggerGen();
+
+            builder.Services.Configure<DatabaseConfiguration>(options =>
+                builder.Configuration.GetSection("DatabaseConfiguration").Bind(options));
+
+            builder.Services.AddScoped<ICustomerService, CustomerService>();
+
+            var app = builder.Build();
+
+            // Configure the HTTP request pipeline.
+            if (app.Environment.IsDevelopment())
+            {
+                app.UseSwagger();
+                app.UseSwaggerUI();
+            }
+
+            app.UseAuthorization();
+            app.MapControllers();
+
+            app.Run();
+        }
+    }
+}
```

Comment:
Use of top level statements would greatly reduce the nesting here and reduce the line count.
https://learn.microsoft.com/en-us/dotnet/csharp/whats-new/tutorials/top-level-statements